astroquery/mast/collections.py

```diff
@@ -43,7 +43,7 @@ def __init__(self):
         services = {"panstarrs": {"path": "panstarrs/{data_release}/{table}.json",
                                   "args": {"data_release": "dr2", "table": "mean"}},
                     "plato": {"path": "plato/{data_release}/{table}.json",
-                                         "args": {"data_release": "dr2", "table": "mean"}}}
+                                         "args": {"data_release": ["dr1", "dr2"], "table": "mean"}}}
```

Comment:
Shouldn't this be panstarrs?

Reply:
Hmm I'm not sure how that snuck in again, I thought I'd removed it. Yes this shouldn't be here at all.

Reply:
I think this is from an older commit because I don't see it in my local branch and I also don't see it under `Files Changed`